doc/source/dev/gitwash/git_links.inc

```diff
@@ -39,14 +36,14 @@
 .. _git branch: https://www.kernel.org/pub/software/scm/git/docs/git-branch.html
 .. _git remote: https://www.kernel.org/pub/software/scm/git/docs/git-remote.html
 .. _git config: https://www.kernel.org/pub/software/scm/git/docs/git-config.html
-.. _why the -a flag?: http://www.gitready.com/beginner/2009/01/18/the-staging-area.html
-.. _git staging area: http://www.gitready.com/beginner/2009/01/18/the-staging-area.html
+.. _why the -a flag?: https://gitready.com/beginner/2009/01/18/the-staging-area.html
+.. _git staging area: https://gitready.com/beginner/2009/01/18/the-staging-area.html
```

Comment:
I noticed that "why the -a flag?" and "git staging area" point to the same link:
https://gitready.com/beginner/2009/01/18/the-staging-area.html

I cannot find this "why the -a flag?" document anywhere, I'll just discard it.